### DISCUSSION TITLE
<ContentDoc /> displaying unexpected results

### DISCUSSION THREAD

**Discussion by dev-adewale** [nuxt/content#1761] [Q&A]
Hey guys 👋, I'm trying to make a Nuxt starter blog template for free use with Nuxt V3 and Content module V2. Everything is working fine until I tried to make a *Catch All Route** Vue file to display individual blog post body.

Below is the code I have in my `[...slug.vue]` file 👇:

```vue
<template>
  <div>
    <ContentDoc />
  </div>
</template>
```

And here is the error I'm getting whenever I tried visiting an individual blog post 👇:

```bash
{
  "message": "You should use slots with <ContentDoc>",
  "slot": "default",
  "data": {
    "_path": "/blog/markdown-post"
  }
}
```

Is there any help?

**Comment by atinux** [comment]
Can you share a reproduction?

**Reply by dev-adewale** [reply]
Hey, thanks for your reply. Pretty sure I'm doing everything right, I just want to understand what the error message I'm getting means.

I shared the repo with you already on Twitter, here it is again. => https://github.com/dev-adewale/nuxt-starter-blog-template

**Reply by dev-adewale** [reply]
I'm successfully pulling content and displaying them on the homepage, but getting the error in the '[...slug.vue]' file.

Here is a link to the file also => https://github.com/dev-adewale/nuxt-starter-blog-template/blob/main/pages/blog/%5B...slug%5D.vue
